lib/presentation/widgets/privacy_policy_and_terms_widget.dart

```diff
@@ -19,19 +17,19 @@ class PrivacyPolicyAndTermsWidget extends StatelessWidget {
           text: LocaleKeys.by_signing_in_you_agree_to_our.tr(),
           style: CustomTheme.restText,
           children: <TextSpan>[
-            spaceTextSpan,
+            CustomTheme.spaceTextSpan,
```

Comment:
you have textSpan in theme? Why??

Reply:
Because you said to take it out, yesterday

Its settings have a color and style, which means their place in the theme